api/DocumentOrShadowRoot.json

```diff
@@ -101,13 +101,10 @@
           }
         }
       },
-      "styleSheets": {
+      "adoptedStyleSheets": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/styleSheets",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/adoptedStyleSheets",
```

Comment:
I notice that https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/adoptedStyleSheets doesn’t exist yet — which is fine and I guess you’re already aware of — but since we don’t (yet) have any way to specify associated spec URLs in BCD (as discussed in #1531), I just want to note that very few other people looking at this data are likely to yet have any idea where `DocumentOrShadowRoot: adoptedStyleSheets` is actually specified (which is in the WICG [Constructable Stylesheet Objects](https://wicg.github.io/construct-stylesheets/#ref-for-dom-documentorshadowroot-adoptedstylesheets%E2%91%A0) proposal).

To be clear: I’m not saying we shouldn’t add stuff like that to BCD — I think it’s fine — I’m just taking the opportunity to say, Gee wouldn’t it be great if when we did, we had a key in the BCD itself to indicate the associated spec.

cc @Elchi3

Reply:
I agree, lets add spec urls soon. :-)